app/javascript/components/recipes/base/selected-ingredient-card.vue

```diff
@@ -79,12 +86,22 @@ export default {
     decreaseQuantity() {
       this.$emit('decrease-quantity', this.recipeIngredientIdx);
     },
+    changeMeasure(measure) {
+      this.$emit('change-measure', measure, this.recipeIngredientIdx);
+    },
   },
   computed: {
     price() {
       return this.recipeIngredientAttrs.ingredientQuantity *
       this.recipeIngredientAttrs.ingredient.price / this.recipeIngredientAttrs.ingredient.quantity;
     },
+    units() {
+      return this.recipeIngredientAttrs.ingredient.otherMeasures.data.map(element => element.attributes.name);
+    },
   },
 };
 </script>
+(element) => ({
+        id: element.id,
+        ...element.attributes,
+      }));
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Newline required at end of file but not found                                eol-last